src/main/java/pl/sda/grocefy/Entity/Item.java

```diff
@@ -0,0 +1,4 @@
+package pl.sda.grocefy.Entity;
```

Comment:
Nazwa pakietu Entity lepiej pisać małą. To takie przyjęte standardy, utarte dla danego języka. Polecam te udostępniane przez Google https://google.github.io/styleguide/javaguide.html#s5.2.1-package-names